README.md

```diff
@@ -0,0 +1,31 @@
+# OpenID Federation 1.0
+
+This repository contains the OpenID Federation specification, which has been migrated from [Bitbucket](https://bitbucket.org/openid/connect/src/master/) to the OpenID Foundation's GitHub.
+The OpenID Federation specification outlines protocols and guidelines for implementing a federated infrastructure across different domains and services.
+
+## Overview
+
+OpenID Federation 1.0 is:
+
+- a technical specification that shows us how to evaluate the Trust from a technical perspective.
+- a technology that shows us how to build trust infrastructures.
+- an advanced system for securely establishing the interoperability of participants.
+
+## Builds
+
+Previews for each branch of this project are automatically built and published at the URL *https://openid.github.io/federation/$branchname.html*.ù
+You can view the editor's copy at [https://openid.github.io/federation/main.html](https://openid.github.io/federation/main.html).
```

Comment:
I suggest adding a link to the latest implementer's draft and latest working group version too.

Reply:
good catch.

Reply:
Done.